pokemongo_bot/cell_workers/handle_soft_ban.py

```diff
@@ -28,7 +28,7 @@ def work(self):
         )
 
         if fort_distance > Constants.MAX_DISTANCE_FORT_IS_REACHABLE:
-            MoveToFort(self.bot).work()
+            MoveToFort(self.bot, self.config).work()
```

Comment:
If MoveToFort takes configuration, this will cause problems since it won't be getting the correct config values. self.config is specific to the task. This is why tasks shouldn't be calling other tasks.

@douglascamata Any ideas on a fix?